src/DynamoRevit/DynamoRevit.cs

```diff
@@ -629,7 +629,7 @@ private static void TryOpenAndExecuteWorkspaceInCommandData(DynamoRevitCommandDa
                     }
                     else
                     {
-                        dynamoViewModel.OpenIfSavedCommand.Execute(new Dynamo.Models.DynamoModel.OpenFileCommand(commandData.JournalData[JournalKeys.DynPathKey], forceManualRun));
+                        dynamoViewModel.OpenCommand.Execute(new Tuple<string,bool>(commandData.JournalData[JournalKeys.DynPathKey], forceManualRun));
```

Comment:
so will this ask the user to save the graph if they have made changes to it? Thats why we added the openIfSavedCommand in the first place for DynamoPlayer usage.

If not, can we modify that command so that it does the right thing with the view?

Reply:
@mjkkirschner Will that case ever happen? How could it happen since player does not allow multiple D4R session?

Reply:
you edit a graph, you make some changes, you don't save, then you click another edit button - I think that was the use case. @BogdanZavu will know for sure.

Reply:
Yes, @mjkkirschner is correct.

Reply:
@mjkkirschner Yes, that's what it was and I also remember there was an issue with the modality of the save dialog which was also fixed by this. @QilongTang Player and Dynamo "share" the same instance of Dynamo/D4R

Reply:
@QilongTang Nope, we always use the existing view model - if there is none we will just start one first time you press edit graph in DP.

Reply:
@BogdanZavu @dobriai @mjkkirschner OK, in that case, I will still use `OpenIfSavedCommand` command and modify the binded function instead